# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/all.js
# Commit: 3eaabde171ef
# Full Hash: 3eaabde171ef2ca02a08da236698327d970bf169
# Author: Tom Ritter <tom@mozilla.com>
# Date: 2021-08-25 21:49:19
# Description:
#   Bug 1727500: Disable crash reporting now that we have some data r=freddyb
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D123624
# ==============================================================================

diff -r 9ad69656cd88 -r 3eaabde171ef modules/libpref/init/all.js
--- a/modules/libpref/init/all.js	Wed Aug 25 21:21:59 2021 +0300
+++ b/modules/libpref/init/all.js	Wed Aug 25 18:19:23 2021 +0000
@@ -214,7 +214,7 @@
 pref("security.osreauthenticator.password_last_changed_hi", 0);
 
 pref("security.crash_tracking.js_load_1.prevCrashes", 0);
-pref("security.crash_tracking.js_load_1.maxCrashes", 1);
+pref("security.crash_tracking.js_load_1.maxCrashes", 0);
 
 pref("general.useragent.compatMode.firefox", false);
 
